com.unity.uiwidgets/package.json

```diff
@@ -1,8 +1,8 @@
 {
 	"name": "com.unity.uiwidgets",
 	"displayName":"UIWidgets",
-	"version": "2.0.3-preview.1",
-	"unity": "2019.4",
+	"version": "2.0.4-preview.1",
+	"unity": "2020.3",
```

Comment:
This indicates the lowest compatible unity version for UIWidgets therefore 2019.4 should be more appropriate. 

Thanks a lot for the kind help @marcuslao !

Reply:
👍